crypto.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import re
+
+from decorators      import is_command
+from crypto_data     import crypto_data
+
+def get_key(value):
+    for key, value in crypto_data.items():
+        if value == value:
+            return key
+
+    return "BTC"
+
+@is_command("Check all the crypto prices.")
+def crypto(message_dict):
+    user = message_dict["display-name"].lower()
+    message = [message_dict["message"]]
+    message_dict["suppress_log"] = True
+    
+    # Attempt to split the message and get each item into an array
+    try:
+        message = re.split(', |; |\s', message)
```

Comment:
Do we need to use regex here? Regex is pretty slow so I usually use `message.split(" ")` in other commands. In addition, `message` contains the full chat message, so the first word is always "!crypto", which will fail to resolve in the dict. 

To get the command's arguments in other commands I use `message.split()[1:]`

Reply:
Large regex functionality is slow, but minor parsing regex isn't any slower than `string.split()`, at least perceptibly.
Really, I plan on pulling out all of the `split()` calls into something like this, just to keep from having so many failures if someone adds a comma or semi-colon instead of just using spaces

Reply:
I prefer string.split() as it's much simpler to read. 

In practise, I doubt that anyone will use commas or semicolons as that's not how chatbots work - and even if they do try something like that, it's just a syntax error and it's fine for the command to return a failure message to chat, or even for the bot to just ignore them. It doesn't need converting to regex project-wide.